### ISSUE TITLE
Copy to clipboard button

### ISSUE THREAD

**Issue by hturner** [r-devel/r-dev-env#95]
The copy to clipboard button that has been added to the markdown files does not work in the VSCode markdown preview, even with changing the security settings so that "Markdown-preview-enhanced: Enable Script Execution" is true (checkbox is checked) and/or running the command "Markdown: Change Preview Security Settings" from the preview and selecting "Disable: Allow all content and script execution".

It may be better to use Javascript to add the button rather than add the button explicitly in the markdown file. Some links that may be helpful:
https://junges.dev/2-how-to-add-github-copy-to-clipboard-button-on-your-docsblog
https://spdevuk.com/how-to-create-code-copy-button/
Then we can decide whether or not to disable scripts in VSCode - if it is disabled at least user don't see a button that doesn't work.

I'm not sure if this issue still applies as I know you had a different style of the docs as work in progress. Perhaps in the short term (for 0.2 release) we can just remove the copy to clipboard buttons?

**Comment by hturner** [comment]
Ah that's even better if the new doc design adds the copy button automatically.

So yes, let's do it like this:

1. Your Doc Design PR - I guess this already removes the buttons and any JS that's no longer needed?
2. My PR making the container README into a tutorial index (https://github.com/r-devel/r-dev-env/pull/94) - I will probably need to resolve conflicts once your Doc Design PR is merged in.

**Comment by hturner** [comment]
Fixed in #98.
